.github/workflows/multichain-e2e-template.yml

```diff
@@ -102,6 +102,13 @@ jobs:
         env:
           FILE: ${{ inputs.config }}
 
+      - name: Capture slog.slog
+        if: always()
+        run: |
+          cd $GITHUB_WORKSPACE
+          echo "Capturing agoric slog.slog"
+          kubectl exec -i agoriclocal-genesis-0 -c validator -- cat slog.slog > "${LOG_DIR}/slog.slog" 2>&1 || true
```

Comment:
Seems like dealer's choice:

```sh
$ time kubectl exec -i agoriclocal-genesis-0 -c validator -- cat slog.slog > test.log
0.24s user 0.21s system 98% cpu 0.457 total

$ time kubectl cp ${NS}/agoriclocal-genesis-0:slog.slog test.log -c validator
tar: slog.slog: file changed as we read it
0.28s user 0.25s system 99% cpu 0.528 total
```

Reply:
hm. yeah... `kubectl cp` seems like what we're doing... a comment about "file changed as we read it" might be nice.

Reply:
I wonder how we do it for a3p.